shopcart.py

```diff
@@ -64,19 +67,108 @@
                 'sku': 114672050,
                 'quantity': 1,
                 'name': "Game of Life",
-                'unitprice': 13.99,
+                'unitprice': 13.99
             }
         ]
     },
 ]
 
+current_shopping_cart_id = 4
+
 ######################################################################
 # GET INDEX
 ######################################################################
 @app.route('/')
 def index():
     return make_response("Hello World", HTTP_200_OK)
 
+######################################################################
+# ADD A NEW SHOPPING CART
+######################################################################
+@app.route('/shopcarts', methods=['POST'])
+def create_shopcarts():
+    payload = request.get_json()
+    if is_validShoppingCart(payload):
+        id = next_sid()
+        if 'products' not in payload:
+            payload['products'] = []
+        shopcart = {'uid': payload['uid'],'sid': id, 'products': payload['products']}
```

Comment:
I think it dones't has to be uid==sid. But still I think we should check if a shopcart exists already?